bigframes/session/__init__.py

```diff
@@ -768,7 +770,7 @@ def _read_gbq_table(
         )
 
         columns = list(columns)
-        filters = list(filters)
+        filters = typing.cast(list, list(filters))
```

Comment:
I'm curious why we need to cast to list here?

Reply:
otherwise mypy complains:

```
bigframes/session/__init__.py:773: error: Incompatible types in assignment (expression has type "list[Iterable[tuple[str, Literal['in', 'not in', '<', '<=', '==', '!=', '>=', '>', 'LIKE'], Any] | str | Any]]", variable has type "Iterable[tuple[str, Literal['in', 'not in', '<', '<=', '==', '!=', '>=', '>', 'LIKE'], Any]] | Iterable[Iterable[tuple[str, Literal['in', 'not in', '<', '<=', '==', '!=', '>=', '>', 'LIKE'], Any]]]")  [assignment]
```